src/Resources/Locales/de_DE.axaml

```diff
@@ -40,13 +40,13 @@
   <x:String x:Key="Text.Archive.Revision" xml:space="preserve">Revision:</x:String>
   <x:String x:Key="Text.Archive.Title" xml:space="preserve">Archiv erstellen</x:String>
   <x:String x:Key="Text.Askpass" xml:space="preserve">SourceGit Askpass</x:String>
-  <x:String x:Key="Text.AssumeUnchanged" xml:space="preserve">UNVERÄNDERTE DATEIEN</x:String>
-  <x:String x:Key="Text.AssumeUnchanged.Empty" xml:space="preserve">KEINE UNVERÄNDERTEN DATEIEN GEFUNDEN</x:String>
+  <x:String x:Key="Text.AssumeUnchanged" xml:space="preserve">ALS UNVERÄNDERT ANGENOMMENE DATEIEN</x:String>
+  <x:String x:Key="Text.AssumeUnchanged.Empty" xml:space="preserve">KEINE UNVERÄNDERT ANGENOMMENEN DATEIEN GEFUNDEN</x:String>
   <x:String x:Key="Text.AssumeUnchanged.Remove" xml:space="preserve">ENTFERNEN</x:String>
   <x:String x:Key="Text.BinaryNotSupported" xml:space="preserve">BINÄRE DATEI NICHT UNTERSTÜTZT!!!</x:String>
   <x:String x:Key="Text.Blame" xml:space="preserve">Blame</x:String>
   <x:String x:Key="Text.BlameTypeNotSupported" xml:space="preserve">BLAME WIRD BEI DIESER DATEI NICHT UNTERSTÜTZT!!!</x:String>
-  <x:String x:Key="Text.BranchCM.Checkout" xml:space="preserve">Checkout ${0}$...</x:String>
+  <x:String x:Key="Text.BranchCM.Checkout" xml:space="preserve">${0}$ auschecken...</x:String>
```

Comment:
I went back and forth about that. Mostly because in the context menu all main commands start with the action "Push", "Pull",... but this would start with the branch name. But I see why you want to change that.

Reply:
Good point I didn't notice that. In that case a consistent design would be preferable. How about "Auschecken auf *master*..." ? It's not perfect but might be a compromise.

Reply:
That's is a compromise. But I would use "**von**" instead of "auf", because it denotes _what branch_ get's checked out and not on what other target something get's checked out ontop. Like in "Was ist das? Das ist der Checkout von 'develop'."
Also this text is used in the context menu for remote and local branches.

Reply:
Yes "von" is indeed more fitting. Like you said it's not on top but just directly navigating there.